developer-docs-site/docs/standards/digital-asset.md

```diff
@@ -336,17 +335,17 @@ are entry functions and do not return any ref (constructor, mutator, etc.). The
 the refs obtained from creating the collection and token objects and do not expose raw access to them.
 
 If a creator wants more custom functionalities such as being able to forcefully transfer a soul bound token, they would
-need to write their own custom module that builds on top of the base token v2 standard. They can of course borrow inspiration
+need to write their own custom module that builds on top of the base Aptos Digital Asset Standard. They can of course borrow inspiration
 and code from the Aptos Token module.
 
 ## Fungible Token
-Similar to [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155), the Token v2 standard also supports fungible tokens
+Similar to [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155), the Aptos Digital Asset Standard also supports fungible tokens
 (also known as semi-fungible tokens). An example of this would be armor tokens in a game. Each armor token represents a
 type of armor and is a token in a collection with metadata (e.g. durability, defense, etc.) and can be minted and burned.
 However, there are multiple instances of the same armor type. For example, a player can have 3 wooden armors, where wooden armor
 is a token in the Armor collection.
 
-This can be easily built by combining Token v2 and Fungible Assets. After the creator creates the Armor collection and the
+This can be easily built by combining Digital Asset (DA) and Fungible Asset (FA) to create Digital and Fungible Asset (DFA). After the creator creates the Armor collection and the
```

Comment:
```suggestion
This can be easily built by creating an asset that is both a Digital Asset (DA) and a Fungible Asset (FA), resulting in a Digital and Fungible Asset (DFA). After the creator creates the Armor collection and the
```

Clarifying a bit here, it's a bit awkward otherwise. Don't mean to nit, just think it could read a little smoother